doc/api/process.md

```diff
@@ -630,7 +630,17 @@ process.
 ```js
 console.log(`Current directory: ${process.cwd()}`);
 ```
+## process.debugPort
+<!-- YAML
+added: v0.7.2
+-->
+* {number}
 
+The port used by Node.js's debugger when enabled.
+
+```js
+  process.debugPort = 5858;
```

Comment:
The linter complains about the indentation. Please do not use any spaces here :-)